### PR TITLE
created header component

### PR DESCRIPTION
**Pull request by pateva**

### CODE REVIEW

File: ics-frontend/src/app/header/header.component.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  .spacer-right {
+    margin-right: 40px; 
+  }
+   
+  .header-right {
+    overflow: hidden;
+    padding: 20px 10px;
+    float: right;
```
Comment (asalov) [inline ics-frontend/src/app/header/header.component.scss:12]: I don't think you need this float

File: ics-frontend/src/app/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<mat-toolbar color="primary">
+    <mat-toolbar-row>
+        <a mat-button routerLink="/image-classification">Tag your image</a>
+        <span class="spacer"></span>
```
Comment (asalov) [inline ics-frontend/src/app/header/header.component.html:4]: It's not a good practice to use such `spacer` elements especially when you can achieve the same effect just by using flexbox.

For example if add the following style declaration
```css
.mat-toolbar-row {
 display: flex;
 justify-content: space-between;
}
```
You can completely remove this `spacer` span (and its styles)

File: ics-frontend/src/app/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        <div class="header-right">
+            <a mat-button routerLink="/image-classification">Analyze</a>
+            <span class="spacer-right"></span>
```
Comment (asalov) [inline ics-frontend/src/app/header/header.component.html:7]: You can remove all of these `spacer-right` spans and achieve the same effect by setting a `gap: 40px` inside the `header-right` styles

File: ics-frontend/src/app/header/header.component.scss (hunk 1 above)
Comment (pateva) [reply ics-frontend/src/app/header/header.component.scss:12]: adjusted

File: ics-frontend/src/app/header/header.component.html (hunk 2 above)
Comment (pateva) [reply ics-frontend/src/app/header/header.component.html:4]: adjusted

**Pull request merged**